ui/raidboss/data/triggers/o5s.js

```diff
@@ -14,48 +54,158 @@
     {
       id: 'O5S Doom Strike',
       regex: /14:28B1:Phantom Train starts using Doom Strike on (\y{Name})/,
+	  regexDe: /14:28B1:Phantomzug starts using Vernichtungsschlag on (\y{Name})/,
+	  //regexFr: /14:28B1:/,
+	  //regexKo: /14:28B1:/,
+	  
       alertText: function(data, matches) {
         if (matches[1] == data.me)
-          return 'Tank Buster on YOU';
+          return {
+			  en: 'Tank Buster on YOU', 
```

Comment:
Can you use two spaces everywhere instead of tabs?

Reply:
i'm editing the files with notepad++ it does the tabs automaticly

Reply:
Settings->Preferences->Language. Tab Settings section.
1. Replace by space: Check the box. 
2. Tab size: 2